ccdproc/core.py

```diff
@@ -703,6 +703,7 @@ def gain_correct(ccd, gain, gain_unit=None):
         gain_value = gain
 
     result = ccd.multiply(gain_value)
+    result.meta = ccd.meta.copy()
```

Comment:
This is also unrelated to the broader change, correct? (agreed it should be fixed) 

Is there a regression test we should add here?